TwinCAT.gitignore

```diff
@@ -0,0 +1,15 @@
+# gitignore template for TwinCAT
+# website: https://www.beckhoff.com/twincat/
+#
+# Recommended: VisualStudio.gitignore
```

Comment:
```suggestion
```

Reply:
You're watching an outdated version. The new version is correctly writing "TwinCAT 3".

Reply:
Rebase the branch